src/core/iomgr/tcp_windows.c

```diff
@@ -207,9 +219,12 @@ static void win_notify_on_read(grpc_endpoint *ep,
 
   /* Did we get data immediately ? Yay. */
   if (info->wsa_error != WSAEWOULDBLOCK) {
+    /* To avoid recursing forever, let's delay the callback for now. */
```

Comment:
I would still prefer to add the comment, but if you think that's contraproductive because the change will come soon, feel free to ignore this.
otherwise LGTM, but check the tests please.